examples/pubsub/tutorial_pubsub_mqtt_publish.c

```diff
@@ -40,13 +40,16 @@
  * options is provided, the system's default certificate location will be used.
  */
 
+#include "open62541/plugin/log_stdout.h"
 #include "open62541/server.h"
 #include "open62541/server_config_default.h"
+
 #include "ua_pubsub.h"
-#include "ua_network_pubsub_mqtt.h"
-#include "open62541/plugin/log_stdout.h"
+
 #include <signal.h>
 
+#include "include/open62541/plugin/pubsub_mqtt.h"
```

Comment:
Remove the leading /include.

Our public headers are wrapped into `< >` braces.